scripts/qa_crowbarsetup.sh

```diff
@@ -1423,14 +1501,22 @@ EOF
                 zypper ar http://${clouddata}/repos/SLES11-SP3-Pool/ sles11sp3
                 zypper ar http://${clouddata}/repos/SLES11-SP3-Updates/ sles11sp3up
             ;;
-            6|7)
+            6)
                 zypper ar http://${clouddata}/repos/SLES12-SP1-Pool/ sles12sp1
                 zypper ar http://${clouddata}/repos/SLES12-SP1-Updates/ sles12sp1up
             ;;
+            7)
+                zypper ar http://${clouddata}/repos/SLES12-SP2-Pool/ sles12sp2
+                zypper ar http://${clouddata}/repos/SLES12-SP2-Updates/ sles12sp2up
+            ;;
         esac
     fi
 
-    if iscloudver 6plus ; then
+
+    if iscloudver 7plus; then
+        onadmin_prepare_sles12sp2_repos
+        onadmin_prepare_sles12plus_cloud_repos
```

Comment:
Don't we need SP1 repos for Cloud7 as well? (for SES). If so, we likely need to adapt also `export_tftpboot_repos_dir`